workflow_ppk.py

```diff
@@ -170,9 +170,10 @@ def main(datadir, geoid, makePos=True, verbose=2, sonar_method='default', rtklib
                 flist_rinex = glob.glob(ff[:-4] + "/*")
                 rover_obs_fname = flist_rinex[np.argwhere([i.endswith('O') for i in flist_rinex]).squeeze()]
                 outfname = os.path.join(os.path.dirname(rover_obs_fname), os.path.basename(flist_rinex[0])[:-3] + "pos")
+                configPath = "configFile/rtkpost.conf"
```

Comment:
I don't love hard coded things, is there a way to set this as default, but also pass as flexible arg?... @kkoetje do we need that?

Reply:
I think this comes up, likely in the next PR we talked about Friday, where we need the config file for each floats set because it will carry location from opus

Reply:
@floydpiper  did we put this as an argument ? or do you want to wait for the next one, i'm fine either way

Reply:
i see the below... asnwers my question